engines/python/setup/djl_python/request_output.py

```diff
@@ -0,0 +1,132 @@
+from dataclasses import field, dataclass
+from typing import List, Optional, Callable, Any, Dict, Union
+
+from djl_python.rolling_batch.rolling_batch import Token
+
+
+@dataclass
+class Sequence:
+    """Dataclass to store the generated tokens details of all the sequences in the request.
+    Attributes:
+        tokens: generated tokens of the sequence.
+        top_tokens: top tokens of the sequence.
+        cumulative_log_prob: cumulative log probability of the sequence.
+        finish_reason: finish reason of the sequence.
+        stop_reason: stop reason of the sequence.
+        generated_text: generated text of the sequence.
+    """
+    tokens: List[Token] = field(default_factory=lambda: [])
+    top_tokens: Optional[List[List[Token]]] = field(default_factory=lambda: [])
+    cumulative_log_prob: float = 0.0
+    finish_reason: str = False
+    stop_reason: Optional[str] = None
+    generated_text: str = ""
```

Comment:
would recommend to keep this as List[str] to store the token text. If customer want to get the full text, we just need to join them

Reply:
Maybe this is not needed, I recalled Token object contains that information